source/IpcModule.ts

```diff
@@ -0,0 +1,57 @@
+import { ipcRenderer, ipcMain } from 'electron'
+
+type Promisified<T extends Function> = 
```

Comment:
Just so I know: TypeScript doesn't support async/await …?

Reply:
It does! The `Promisfied` stuff here is only for convince: if the original method (in the interface) returns "string" then the "ipcfied" method returns "Promise<string>" (essentially since `ipcRenderer.invoke` returns a promise. These type declarations here take care of this automatic conversion.